src/odemis/driver/test/xt_client_test.py

```diff
@@ -712,6 +711,72 @@ def test_apply_autofocus(self):
         autofocus_state = self.microscope.is_autofocusing(channel)
         self.assertEqual(autofocus_state, False)
 
+    def test_get_mpp_orientation(self):
+        """
+        Simple test, checks if no error occur during calling the method and the return value.
+        """
+        mpp_orientation = self.microscope.get_mpp_orientation()
+        self.assertIsInstance(mpp_orientation, float)
+        # TODO K.K. check range with Wilco
+        self.assertTrue(0 < mpp_orientation < 360)
+
+    def test_get_aperture_index(self):
+        """
+        Simple test, checks if no error occur during calling the method and the return value.
+        """
+        aperture_index = self.microscope.get_aperture_index()
+        self.assertIsInstance(aperture_index, float)
+        self.assertTrue(0 < aperture_index < 14)
+
+    def test_set_aperture_index(self):
+        """
+        Simple test, checks if no error occur during calling the method.
+        """
+        # TODO Check if setting of the value actually happens and if an int value is also allowed.
```

Comment:
You can remove this TODO here. This indeed needs to be checked. But does not need to be noted here. Radim is working on the TFS implementation to actually return an error in case an aperture cannot be set.